src/LiveDevelopment/MultiBrowserImpl/transports/remote/PostMessageTransportRemote.js

```diff
@@ -0,0 +1,61 @@
+// This is a transport injected into the browser via a script that handles the low
+// level communication between the live development protocol handlers on both sides.
+// This transport provides a postMessage mechanism. It's injected separately from the
+// protocol handler so that the transport can be changed separately.
+
+(function (global) {
+    "use strict";
+    
+   var postMessageTransport = {
```

Comment:
The indenting is all wonky here and below - make sure to compare it to the NodeSocketTransportRemote